src/js/error-summary.js

```diff
@@ -17,6 +17,12 @@ class ErrorSummary {
 	*/
 	constructor(elements) {
 		this.elements = elements;
+		const hasAnInverseField = elements.some(elem => {
+			if (elem.field) {
+				return elem.field.classList.contains('o-forms-field--inverse');
+			}
+		});
+		this.inverse = hasAnInverseField;
```

Comment:
I missed this in the original implementation. We need to ensure that the error summary created by this class uses the correct design. I.E. an inverse design for an inverse form.

The way I've done that is to check if any form field has an inverse modifier class and if any do, then the error summary will be inverse style.